examples/hotrod/kubernetes/README.md

```diff
@@ -0,0 +1,9 @@
+# Hot R.O.D. - Rides on Demand on Kubernetes
+
+Example k8s manifests for deploying the [hotrod app](..) to your k8s environment of choice. e.g. minikube, k3s, EKS, GKE
+
+## Deploy with Kustomize
+
+```
+kustomize build . | kubectl apply -f -
```

Comment:
there are more steps in the top README

Reply:
Copied over steps from top README with modified paths.